.swiftlint.yml

```diff
@@ -16,6 +15,7 @@ opt_in_rules: # some rules are opt-in only
 - toggle_bool
 - unneeded_parentheses_in_closure_argument
 - vertical_parameter_alignment_on_call
+# - missing_docs
```

Comment:
Where was this rule giving you trouble? Are you able to just add some docs without removing the rule?